src/database/Storage.js

```diff
@@ -0,0 +1,22 @@
+const { Storage } = require('@google-cloud/storage');
+const storage = new Storage();
+const bucket = storage.bucket('recipe_photos');
+

```

Comment:
for yourself later on, add a comment per file describing what the file is meant to do.

e.g. here: "handles image uploading for a recipe"

Reply:
done